src/user-guide/writing-workflows/index.rst

```diff
@@ -5,11 +5,11 @@ Writing Workflows
    :maxdepth: 2
 
    configuration
-   scheduler
    scheduling
    runtime
    parameterized-tasks
    jinja2
    empy
    external-triggers
    suicide-triggers
+   scheduler
```

Comment:
This is more admin docs than user docs, when following section order (i.e. clicking the next button) this comes first after the config section which is confusing so I've bumped it to the end.